templates/docker-entrypoint.sh

```diff
@@ -94,6 +94,20 @@ if  [[ "$1" == apache2* || "$1" == php-fpm || "$1" == bin* ]]; then
       ${ROUNDCUBEMAIL_PLUGINS_SH};
   fi
 
+  if [ ! -d skins/${ROUNDCUBEMAIL_SKIN} ]; then
```

Comment:
Add `echo` like above for plugins installation. Just to be consistent.

Reply:
Done